README.md

```diff
@@ -1,6 +1,43 @@
-# Setup
+# Dashi
+
+Dashi is a quality control reporting system developed for genomic and
+transcriptomic data. It is built on top of
+[gsi-qc-etl](https://bitbucket.oicr.on.ca/projects/GSI/repos/gsi-qc-etl/browse),
+OICR's private QC data integration system, and so is currently of limited
+utility to external parties. Please contact us for more information.
+
+## Setup on bare metal
+
 1. Install python3, pip
 1. Setup new virtual environment
 1. `pip install -r requirements.txt`
 1. Set the GSI_ETL_CACHE_DIR environment variable
-1. `flask run` **OR** `gunicorn --bind 0.0.0.0:5000 wsgi:app`
\ No newline at end of file
+1. `flask run` **OR** `gunicorn --bind 0.0.0.0:5000 wsgi:app`
+
+
+## Set up Docker container
+
+The Docker container is a more straightforward way to launch Dashi for testing.
+The base container has most of the dependencies required for the app, with one
+crucial exception: gsi-qc-etl. This repository is private to OICR and so you
+need to pass in your SSH keys to permit download and installation.
+
+**Requirements**
+* Docker 18.09.6+
+* Docker compose 1.23.1+
+* Access to [gsi-qc-etl](https://bitbucket.oicr.on.ca/projects/GSI/repos/gsi-qc-etl/browse)
+
+
+1. Ensure your ssh key has been added to OICR's Bitbucket and you can access and
+   clone
+   [gsi-qc-etl](https://bitbucket.oicr.on.ca/projects/GSI/repos/gsi-qc-etl/browse).
+2. Download the gsi-qc-etl cache data to `cache_files` (or modify
+   docker-compose.yml to point to the correct location). The current location for
+   this on OICR's cluster is at
+   `/.mounts/labs/mcphersonlab/public/slazic/2019-01-16-GSI_ETL/dev_cache_files`.
+3. Build the container with `docker-compose build`. Note that this completes
```

Comment:
I tried to build this locally and ran into an error:
```
  git@bitbucket.oicr.on.ca: Permission denied (publickey).
  fatal: Could not read from remote repository.

  Please make sure you have the correct access rights and the repository exists.
```
I tried adding my key to the "Access Keys" tab on Bitbucket and was told that this key was already associated with user harmstrong...

Reply:
You don't want to add it to Access keys. That's for machines. Can you clone that repository with the ssh url?

Reply:
Yes, I'm able to clone with the ssh URL.

Reply:
We figured out that the issue arose because @callunity had a passphrase on her SSH key. Will need to figure out how to pass that in.

Reply:
There's instructions in the README and compose on how to use an SSH key with a passphrase.